### ISSUE TITLE
Remove legacy tags.

### ISSUE THREAD

**Issue by ypid** [debops/debops-playbooks#211]
This command comes in handy. Just tried it and changes:

```diff
     - role: ypid.apparmor
-      tags: [ 'role::apparmor', 'apparmor' ]
+      tags: [ 'role::apparmor' ]
```

```Shell
git ls-files | xargs sed --in-place --regexp-extended "s/(tags: \[ 'role::[^']+?'), .* ]/\1 ]/"
```

**Comment by drybjed** [comment]
Thanks, good to know. I'll probably remove legacy tags in the near future when more roles are updated with their dependencies moved to playbooks.

**Comment by ypid** [comment]
Fixed by #212. Hope the regex magic helped :wink:

**Comment by drybjed** [comment]
@ypid It sure did, a bit. Still had to clean up the `- hosts:` lines by hand though.
